lizzy_client/utils.py

```diff
@@ -1,11 +1,37 @@
-from collections import namedtuple
+import os
 import re
-import yaml
+from collections import namedtuple
+from urllib.error import URLError
+from urllib.parse import quote
+from urllib.request import urlopen
 
+import click
+import yaml
 
 StackReference = namedtuple('StackReference', 'name version')
 
 
+def read_parameter_file(parameter_file):  # copy pasted from Senza
+    paras = []
+
+    try:
+        url = (parameter_file if '://' in parameter_file
+               else 'file://{}'.format(quote(os.path.abspath(parameter_file))))
+
+        response = urlopen(url)
+    except URLError:
+        raise click.UsageError('Can\'t read parameter file "{}"'.format(parameter_file))
+
+    try:
+        cfg = yaml.safe_load(response.read())
+        for key, val in cfg.items():
+            paras.append("{}={}".format(key, val))
+    except yaml.YAMLError as e:
+        raise click.UsageError('Error {}'.format(e))
+
+    return tuple(paras)
```

Comment:
This was copied from senza. In senza it's a tuple because the cli parameters are received as a tuple, and the file parameters are appended with `parameter = parameter + parameter_from_file`, but in lizzy I convert the parameters to a list and extend the list to make it clear it's mutable. I'm going to remove the conversion to tuple here.